qtconsole/tests/test_ansi_code_processor.py

```diff
@@ -49,9 +50,15 @@ def test_colors(self):
             elif i == 2:
                 self.assertEqual(substring, 'last')
                 self.assertEqual(self.processor.foreground_color, None)
+            elif i == 3:
+                foreground_color = self.processor.foreground_color
+                self.assertEqual(self.qt_processor.get_color(foreground_color).name(), '#ffd700')
+                self.assertEqual(substring, 'Yellow')
+                self.assertEqual(self.processor.foreground_color, 3)
```

Comment:
Just for clarity, maybe could be worthy to write this like:

```suggestion
                foreground_color = self.processor.foreground_color
                self.assertEqual(substring, 'Yellow')
                self.assertEqual(foreground_color, 3)
                self.assertEqual(self.qt_processor.get_color(foreground_color).name(), '#ffd700')
                
```